modules/apps/dispatch/dispatch-test/src/testIntegration/java/com/liferay/dispatch/service/test/DispatchTriggerLocalServiceTest.java

```diff
@@ -636,6 +673,34 @@ private String _getRandomDispatchExecutorType() {
 		return TestDispatchTaskExecutor.DISPATCH_TASK_EXECUTOR_TYPE_TEST;
 	}
 
+	private void _testUpdateDispatchTriggerWithCronExpressions(
+			DispatchTrigger dispatchTrigger,
+			DispatchTaskClusterMode dispatchTaskClusterMode,
+			Calendar startCalendar, String cronExpression)
+		throws Exception {
+
+		Calendar endCalendar = CalendarFactoryUtil.getCalendar(
+			startCalendar.getTimeInMillis());
+
+		endCalendar.add(Calendar.DAY_OF_YEAR, 7);
+
+		dispatchTrigger = _dispatchTriggerLocalService.updateDispatchTrigger(
+			dispatchTrigger.getDispatchTriggerId(), true, cronExpression,
+			dispatchTaskClusterMode, endCalendar.get(Calendar.MONTH),
+			endCalendar.get(Calendar.DAY_OF_MONTH),
+			endCalendar.get(Calendar.YEAR), 23, 59, false, true,
+			startCalendar.get(Calendar.MONTH),
+			startCalendar.get(Calendar.DAY_OF_MONTH),
+			startCalendar.get(Calendar.YEAR), 23, 59, "UTC");
+
+		Thread.sleep(1500);
+
+		Assert.assertEquals(
```

Comment:
I don't understand this assert. For all the tests, we are asserting the same? That is weird